src/main/java/net/mcreator/element/types/BannerPattern.java

```diff
@@ -93,4 +90,27 @@ public String bannerDescription(String color) {
 	@Override public ImageIcon getIconForMCItem(Workspace workspace, String suffix) {
 		return workspace.getFolderManager().getTextureImageIcon(texture, TextureType.ITEM);
 	}
+
+	@Override public List<ColorEntry> colorEntries() {
+		return List.of(new BannerPattern.ColorEntry("blue", description),
+				new BannerPattern.ColorEntry("white", description), new BannerPattern.ColorEntry("orange", description),
+				new BannerPattern.ColorEntry("magenta", description),
+				new BannerPattern.ColorEntry("light_blue", description),
+				new BannerPattern.ColorEntry("yellow", description), new BannerPattern.ColorEntry("lime", description),
+				new BannerPattern.ColorEntry("pink", description), new BannerPattern.ColorEntry("gray", description),
+				new BannerPattern.ColorEntry("light_gray", description),
+				new BannerPattern.ColorEntry("cyan", description), new BannerPattern.ColorEntry("purple", description),
+				new BannerPattern.ColorEntry("brown", description), new BannerPattern.ColorEntry("green", description),
+				new BannerPattern.ColorEntry("red", description), new BannerPattern.ColorEntry("black", description));
+	}
+
+	public record ColorEntry(String color, String description) {
+		public String getColor() {
```

Comment:
You can replace usages of this method with `color()` method provided by the record signature.